Help upgrading to Hugo v0.143.1

The site does not build with Hugo v0.143.1
```
$ make develop
HUGO_PREVIEW=true hugo server \
		--buildDrafts \
		--buildFuture \
		--ignoreCache
Error: error building site: render: failed to render pages: render of "/" failed: "/Users/.../jaegertracing/documentation/themes/jaeger-docs/layouts/index.html:4:10": execute of template failed: template: index.html:4:10: executing "main" at <.Err>: can't evaluate field Err in type resource.Resource: Resource.Err was removed in Hugo v0.141.0 and replaced with a new try keyword, see https://gohugo.io/functions/go-template/try/
make: *** [develop] Error 1
```

--- a/.github/workflows/ci-test.yml
+++ b/.github/workflows/ci-test.yml
@@ -17,7 +17,7 @@ jobs:
     - name: ✨ Setup Hugo
       env:
         # should match one from netlify.toml
-        HUGO_VERSION: 0.123.6
+        HUGO_VERSION: 0.143.1
       run: |
         mkdir ~/hugo
         cd ~/hugo

--- a/netlify.toml
+++ b/netlify.toml
@@ -3,6 +3,7 @@ publish = "public"
 command = "make netlify-production-build"
 
 [build.environment]
+# should match one from .github/workflows/ci-test.yml
 HUGO_VERSION = "0.143.1"
 
 [context.deploy-preview]

--- a/netlify.toml
+++ b/netlify.toml
@@ -3,7 +3,7 @@ publish = "public"
 command = "make netlify-production-build"
 
 [build.environment]
-HUGO_VERSION = "0.123.6"
+HUGO_VERSION = "0.143.1"
 
 [context.deploy-preview]
 command = "make netlify-deploy-preview"

--- a/themes/jaeger-docs/layouts/index.html
+++ b/themes/jaeger-docs/layouts/index.html
@@ -1,10 +1,12 @@
 {{ define "main" }}
-{{ $rss := "" }}
-{{ with resources.GetRemote "https://medium.com/feed/jaegertracing" }}
+{{ $rss := dict }}
+{{ with try (resources.GetRemote "https://medium.com/feed/jaegertracing") }}
   {{ with .Err }}
-    {{ errorf "%s" . }}
-  {{ else }}
+    {{ warnf "%s" . }}
+  {{ else with .Value }}
     {{ $rss = . | transform.Unmarshal }}
+  {{ else }}
+    {{ warnf "Unable to fetch RSS feed" }}
   {{ end }}
 {{ end }}
 

--- a/themes/jaeger-docs/layouts/index.html
+++ b/themes/jaeger-docs/layouts/index.html
@@ -2,7 +2,7 @@
 {{ $rss := dict }}
 {{ with try (resources.GetRemote "https://medium.com/feed/jaegertracing") }}
   {{ with .Err }}
-    {{ warnf "%s" . }}
+    {{ errorf "%s" . }}
   {{ else with .Value }}
     {{ $rss = . | transform.Unmarshal }}
   {{ else }}

